src/conn/conn_stat.c

```diff
@@ -76,6 +76,7 @@ __wt_conn_stat_init(WT_SESSION_IMPL *session)
     __wti_cache_stats_update(session);
     __wt_evict_stats_update(session);
     __wt_txn_stats_update(session);
+    __wt_checkpoint_stats_time_update(session);
```

Comment:
Alphabetize with the other callers. I'd move it up a couple lines to just below `wti_cache_stats_update`.

Reply:
Sure, done in 800335826a8f553c758d7e9b5f0409f2486a50aa